Cannot connect the database in PostgreSQL 13 and received this error message in the python file. Please let me know if you have any suggestions on correcting my work.

Thank you very much.
